pkg/transformers/image.go

```diff
@@ -91,7 +90,7 @@ func (pt *imageTransformer) findAndReplaceImage(obj map[string]interface{}) erro
 func (pt *imageTransformer) updateContainers(in interface{}) (interface{}, error) {
 	containers, ok := in.([]interface{})
 	if !ok {
-		return nil, fmt.Errorf("containers path is not of type []interface{} but %T", in)
+		return nil, nil
```

Comment:
This works but may swallow some user errors for invalid configuration. I would prefer to relax the check in `findAndReplaceImage` instead.

Reply:
I agree that this change permits a greater range of configuration variations.
I will close this PR, as you have taken on the issue.